usr/share/rear/conf/GNU/Linux.conf

```diff
@@ -164,6 +164,24 @@ sysctl
 blockdev
 lsblk
 clear
+
+# Older releases of os-prober (1.74 and below) use dmsetup as a fallback
+# solution for mounting when grub-mount is missing.
+#
+# However, dmsetup was included in the rescue image if and only if LVM,
+# multipath or encryption were detected.  Thus, BIOS machines that do
+# not use these but still have dmsetup present, would block indefinitely
+# on the "Installing GRUB2 boot loader..." step.
+#
+# GRUB2 installation is performed in a chroot after the data have already
+# been recovered.  ReaR would call grub-mkconfig which calls os-prober
+# which then executes dmsetup.  However, it would never receive the response
+# trough host's udev as the rescue system would not have dmsetup present
```

Comment:
*through, and while at it you can say that the expected response is in the form of releasing a SysV semaphore by dmsetup executed from udev and reference the related debian-installer bugs.

Reply:
thinking about it, the response is more "from udev processing" than "through udev" as it bypasses the udev control socket and is sent directly from one dmsetup to another via sysvipc.